src/components/radio-group/radio-button.tsx

```diff
@@ -62,6 +66,18 @@ export class RadioButton extends React.Component<RadioButtonProps, RadioButtonSt
         );
     }
 
+    private handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
There's also a special type signature for event handlers:
`private handleClick: React.MouseEventHandler<HTMLElement> = e => {`